routes/users.js

```diff
@@ -1,9 +1,94 @@
 const express = require('express');
+const { userWidgets, db } = require('../database/schema.js')
 const router = express.Router();
+const mongoose = require('mongoose')
+
 
 /* GET users listing. */
-router.get('/', (request, response, next) => {
-  response.send('respond with a resource');
+router.get('/', ( request, response, next ) => {
+  response.send( 'respond with a resource' );
+});
+
+/* GET widget */
+router.get('/widgets/:widgetId', ( request, response, next ) => {
```

Comment:
I would name this parameter `:id` - the URL makes it explicit that this is a widget's id.